src/Bot.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rubi\RubiBot;
+
+use GuzzleHttp\Client;
+
+class Bot
+{
+    private string $boturl;
+    private Client $client;
+    public int $chatId;
+
+    public function __construct(string $token)
+    {
+        $this->boturl = "https://api.telegram.org/bot$token/";
+        $this->client = new Client([
+            'base_uri' => $this->boturl,
+        ]);
+

```

Comment:
Da wir jetzt den Client haben brauchen wir die boturl gar nicht mehr einzeln und können uns diese property sparen

```suggestion
    private Client $client;
    public int $chatId;

    public function __construct(string $token)
    {
        $this->client = new Client([
            'base_uri' => "https://api.telegram.org/bot$token/"
        ]);


```